src/quacc/recipes/espresso/core.py

```diff
@@ -350,3 +350,60 @@ def non_scf_job(
         additional_fields={"name": "pw.x Non SCF"},
         copy_files=prev_dir,
     )
+
+
+@job
+def bands_job(
+    atoms: Atoms,
+    prev_dir: str | Path,
+    preset: str | None = "sssp_1.3.0_pbe_efficiency",
+    parallel_info: dict[str] | None = None,
+    test_run: bool = False,
+    **calc_kwargs,
+) -> RunSchema:
+    """
+    Function to carry out a basic bands calculation with pw.x.
```

Comment:
Might be helpful to the user to mention that this is a non-self consistent calculation.